.prettierignore

```diff
@@ -2,4 +2,9 @@
 src/emoji/__tests__/data-test.js
 
 # We're not allowing Prettier to touch some of our vendored code.
-src/third/redux-persist
\ No newline at end of file
+src/third/redux-persist
+
+# We'd like to keep the original formatting for some Flow libdefs from
+# FlowTyped, to make it easier to eventually make a PR back to
+# FlowTyped.
```

Comment:
ESLint already ignores them because of the `**/__flow-tests__/**`
    line in `.eslintignore`.

nit: I think you mean the other line, `**/flow-typed/**`

Reply:
Indeed, thanks.